src/radiation_system.hpp

```diff
@@ -28,10 +28,15 @@
 #include "hyperbolic_system.hpp"
 #include "simulation.hpp"
 #include "valarray.hpp"
+#include "planck_integral.hpp"
 
 // physical constants in CGS units
 static constexpr double c_light_cgs_ = 2.99792458e10;	      // cgs
 static constexpr double radiation_constant_cgs_ = 7.5646e-15; // cgs
+static constexpr double pi = 3.14159265358979323846;
+static constexpr double hplanck = 6.62606957e-27;  // erg s
```

Comment:
We din't need to define these again in quokka, we can just call the ones already defined in Microphysics. For example, k_B can be called as `C::k_B`

Reply:
Yes, good catch. We should change all of these to just use the values from Microphysics. They are all in the `C` namespace (C is for "constants").

Reply:
See this file for the available constants: https://github.com/AMReX-Astro/Microphysics/blob/main/constants/fundamental_constants.H